src/table/__integ__/resizable-columns.test.ts

```diff
@@ -91,26 +91,20 @@ class TablePage extends BasePageObject {
   async resizeBeyondTableWidth(columnIndex: number) {
     const resizerSelector = tableWrapper.findColumnResizer(columnIndex).toSelector();
     const resizerBox = await this.getBoundingBox(resizerSelector);
-    const { width: windowWidth } = await this.browser.getWindowSize();
     await this.browser.performActions([
       {
         type: 'pointer',
         id: 'mouse',
         parameters: { pointerType: 'mouse' },
         actions: [
           // hover over resizer
-          {
-            type: 'pointerMove',
-            duration: 0,
-            x: prepareCoordinate(resizerBox.left),
-            y: prepareCoordinate(resizerBox.top),
-          },
+          { type: 'pointerMove', x: Math.round(resizerBox.left), y: Math.round(resizerBox.top) },
           { type: 'pointerDown', button: 0 },
-          { type: 'pause', duration: 30 },
+          { type: 'pause', duration: 50 },
           // move cursor to screen edge to activate auto-growing behavior
-          { type: 'pointerMove', duration: 0, x: windowWidth, y: 0 },
+          { type: 'pointerMove', duration: 10, x: Math.round(resizerBox.left) + 15, y: Math.round(resizerBox.top) },
```

Comment:
Why was `windowWidth` removed? Now it is a very different test

Reply:
I removed it as part of my experiments, that also allows to avoid querying the window width.

That does not make the test any different though.

Reply:
The comment says: "move cursor to screen edge". `x: windowWidth` is definitely a screen edge. For `x: Math.round(resizerBox.left) + 15` is it unclear. Even if it coincidentally corresponds to the screen edge , nobody promises you that.

Reply:
Alright, I revert this bit and re-validate.

Reply:
Does the same comment apply to the change to the change in y co-ordinate?

Reply:
The `y` coordinate should be irrelevant here: only horizontal movement matter. I replaced 0 with resizer coordinate as it represents the expected user interaction better.